src/Data/MeasuredIn.hs

```diff
@@ -0,0 +1,119 @@
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE DeriveGeneric #-}
+{-# LANGUAGE DerivingStrategies #-}
+{-# LANGUAGE FlexibleInstances #-}
+{-# LANGUAGE GeneralizedNewtypeDeriving #-}
+{-# LANGUAGE KindSignatures #-}
+{-# LANGUAGE LambdaCase #-}
+{-# LANGUAGE Rank2Types #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE StandaloneDeriving #-}
+{-# LANGUAGE TypeApplications #-}
+
+-- |
+-- Copyright: © 2018-2019 IOHK
+-- License: MIT
+--
+-- Representation of values with an associated (free) unit of measure. Useful to
+-- disambiguate primitive types like 'Int' or 'String' which can be in different
+-- bases depending on the context.
+
+module Data.MeasuredIn
+    ( -- * Polymorphic MeasuredIn
+      MeasuredIn(..)
```

Comment:
Perhaps this is just nitpicking, but some of this styling seems inconsistent.

Some brackets have spaces before them, but others don't.

Given that we already have a well-defined style for imports (== spaces before parentheses), perhaps we can copy that style for exports? (Can make a proposal if you like. )

Reply:
I'd say, if we can get stylish-haskell or hlint to do it for us, definitely. Otherwise, that will rapidly become annoying to chase those :/ 
I do like the consistency here however.